webrtc.html

```diff
@@ -5536,6 +5538,19 @@ <h2>Attributes</h2>
               <p>On getting, the attribute MUST return the value of the
               <a>[[\SenderRtcpTransport]]</a> slot.</p>
             </dd>
+            <dt><dfn><code>mstId</code></dfn> of type <span class=
+            "idlAttrType"><a>DOMString</a></span>, readonly, nullable</dt>
+            <dd>
+              <p>MediaStreamTrack id that appears in the msid attribute of the
+              related SDP mline or null if there is not msid attribute.
+              Useful for senders created without a track (or detached from a track).</p>
```

Comment:
Exactly how would this be useful?

Reply:
There should be some use cases where this is quite useful.
I am exposing why we needed to implement that:
- Before having any track, you perform addTransceiverByKind(), so an RtpTransceiver with an RtpSender is created
- An SDP offer is created and signaled to the peer attaching some application metadata (needed for APP functionality) related to the MediaStreamTrack id being in the SDP a=msid attr.
- Then the a track is attached to the RtpSender.
- In addition, the track (or others tracks) can be detached and attached again and again.